docs/technical/settings.rst

```diff
@@ -0,0 +1,19 @@
+Settings
+========
+
+This is the list of Django settings defined in the ``common.py`` modules of edx-platform.
+
+.. note::
+    Toggle settings, which enable or disable a specific feature, are more extensively documented in the :ref:`feature toggles <featuretoggles>` section.
+
+LMS settings
+------------
+
+.. settings::
+    :folder_path: lms/envs/common.py
```

Comment:
We’ll have to see if this works or not. What if a Django app uses a setting with a default and doesn’t need it to be defined in common.py?

Reply:
My opinion on this is that settings should always have a default -- the only exception being the one we discussed earlier, which are feature toggle settings.

Reply:
👍  That's a reasonable place to start at least.